Fix vale warnings in End-user Guide (#2078)
codenvy/che-docs: modules/end-user-guide/partials/con_che-theia-plug-in-concept-in-detail.adoc

[id="che-theia-plug-in-apis_{context}"]
== Che-Theia plug-in APIs

> For the purpose of providing tool isolation and easy extensibility in {prod}, the Che-Theia IDE has a set of plug-in APIs. The APIs are compatible with Visual Studio Code extension APIs. Usually, Che-Theia can run VS Code extensions as its own plug-ins.

Reviewer: ```suggestion
To provide tool isolation and easy extensibility in {prod}, the Che-Theia IDE has a set of plug-in APIs. The APIs are compatible with Visual Studio Code extension APIs. Usually, Che-Theia can run VS Code extensions as its own plug-ins.
```
